src/components/Book.js

```diff
@@ -4,22 +4,54 @@ import PropTypes from 'prop-types';
 function Book({ bookObject, deleteBook }) {
   /* eslint-disable no-console */
   return (
-    <tbody>
-      <tr>
-        <td>{bookObject.id}</td>
-        <td>{bookObject.title}</td>
-        <td>{bookObject.category}</td>
-        <td>
-          <button
+    <div id="book">
+      <div className="book-info">
+        <p className="book-category">{bookObject.category}</p>
+        <h3 className="book-title">{bookObject.title}</h3>
+        <p className="book-author">Stan Lee</p>
+        <div className="info-edit">
+          <span>Comments</span>
+          <div
+            className="removeBtn"
             type="button"
             onClick={() => deleteBook(bookObject.id)}
-            style={{ color: 'tomato' }}
+            aria-hidden="true"
           >
-            Delete
-          </button>
-        </td>
-      </tr>
-    </tbody>
+            Remove
+          </div>
+          <span>Edit</span>
+        </div>
+      </div>
+      <div className="book-percent">
+        <div className="percent-circle" />
+        <div className="percent-text">
+          <p className="percent-num">0%</p>
+          <p className="percent-completed">Completed</p>
+        </div>
+      </div>
+      <div className="book-chapters">
+        <p className="cur-chapter-h">CURRENT CHAPTER</p>
+        <p className="cur-chapter">Chapter 1</p>
+        <button className="cur-chapter-btn" type="button">UPDATE PROGRESS</button>
+      </div>
+    </div>
+
+  // <tbody>
+  //   <tr>
+  //     <td>{bookObject.id}</td>
+  //     <td>{bookObject.title}</td>
+  //     <td>{bookObject.category}</td>
+  //     <td>
+  //       <button
+  //         type="button"
+  //         onClick={() => deleteBook(bookObject.id)}
+  //         style={{ color: 'tomato' }}
+  //       >
+  //         Delete
+  //       </button>
+  //     </td>
+  //   </tr>
+  // </tbody>
```

Comment:
It’s better to delete **commented pieces of code**. If you need to review a previous version of your code, use git history ([screenshot](https://gitlab.com/microverse/guides/projects/requirements_screenshots/raw/master/images/general/no_inline_comments.png) from [Pathwright](https://microverse.pathwright.com/library/fast-track-curriculum/69047/path/step/66243643/))

Reply:
Thanks a lot for the suggestions @trekab
All done  :+1: 

Kind regards